src/components/CurrencyInputPanel/CurrencyInputPanel.container.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useEffect, useMemo } from 'react'
+import debounce from 'lodash.debounce'
```

Comment:
```suggestion
import debounce from 'lodash/debounce';
```

Recommend to add whole lodash to the repository and use it like this instead of adding each one to package.json. This will be tree shaked by webpack